proposals/NNNN-vk-sampled-texture.md

```diff
@@ -0,0 +1,184 @@
+<!-- {% raw %} -->
+
+# Vulkan Sampled Textures
+
+* Proposal: [NNNN](NNNN-vk-sampled-texture.md)
+* Author(s): [Cassandra Beckley](https://github.com/cassiebeckley)
+* Sponsor: [Steven Perron](https://github.com/s-perron) and TBD
+* Status: **Under Consideration**
+* Required Version: HLSL 202x
```

Comment:
Per our discussion:

Marking the attribute that you want to remove deprecated in HLSL 2021, then remove it in HLSL 202x seems like a good path forward.

For the new types, we can use a `__has_feature` check in the preprocessor and make it available in all language modes.